core/src/main/java/org/apache/spark/unsafe/map/BytesToBytesMap.java

```diff
@@ -221,7 +221,7 @@ public BytesToBytesMap(
       SparkEnv.get() != null ? SparkEnv.get().blockManager() :  null,
       SparkEnv.get() != null ? SparkEnv.get().serializerManager() :  null,
       initialCapacity,
-      0.70,
+      0.5,
```

Comment:
Should we make this a constant and document why it was chosen (to enable radix sort)?